pallets/dao/src/weights.rs

```diff
@@ -0,0 +1,47 @@
+use frame_support::weights::Weight;
+
+pub trait WeightInfo {
+	fn create_vision() -> Weight;
+	fn remove_vision() -> Weight;
+	fn sign_vision() -> Weight;
+	fn unsign_vision() -> Weight;
+	fn create_organization() -> Weight;
+	fn dissolve_organization() -> Weight;
+	fn add_members() -> Weight;
+	fn add_tasks() -> Weight;
+	fn remove_members() -> Weight;
+	fn remove_tasks() -> Weight;
+}
+
+impl WeightInfo for () {
+	fn create_vision() -> Weight {
```

Comment:
Why is the weights hardcoded here? Also, we have weights for dao in 

- pallets/dao/src,rs
- runtime/src/weights/dao.rs
- runtime/src/weights/pallet_dao.rs

Isn't there a better way to organize this?